cmd/queue/main.go

```diff
@@ -16,8 +16,14 @@ limitations under the License.
 
 package main
 
-import "knative.dev/serving/pkg/queue/sharedmain"
+import (
+	"os"
+
+	"knative.dev/serving/pkg/queue/sharedmain"
+)
 
 func main() {
-	sharedmain.Main()
+	if sharedmain.Main() != nil {
+		os.Exit(1)
+	}
```

Comment:
we should also probably log the error here

Reply:
Since we let downstream change this file, I think it is best if we log inside sharedmain to ensure consistent logging. It seems better to log inside sharedmain.